lua/lspconfig/configs/texlab.lua

```diff
@@ -91,9 +95,11 @@ local function command_factory(cmd)
 end
 
 local function buf_find_envs(client, bufnr)
+  local win = vim.api.nvim_get_current_win()
+  local params = vim.lsp.util.make_position_params(win, client.offset_encoding)
   client.request('workspace/executeCommand', {
     command = 'texlab.findEnvironments',
-    arguments = { vim.lsp.util.make_position_params() },
+    arguments = { params },
```

Comment:
```suggestion
    arguments = { vim.lsp.util.make_position_params(win, client.offset_encoding) },
```

Reply:
params a bit confusing here inside arguments and inline would be better

Reply:
Done.